CesiumGltf/generated/include/CesiumGltf/Material.h

```diff
@@ -12,6 +12,9 @@
 #include <optional>
 #include <vector>
 
+// OPAQUE is defined in wingdi.h
+#undef OPAQUE
```

Comment:
this turns out to be quite cumbersome to use the glTF library in O3DE too. I think we should rename the enum to something else (Maybe add prefix gltf to enum)

Reply:
or maybe use `Opaque` as a name with the first letter capitalized (Not entirely fool-proofed for what window does, but seems to be good enough)

Reply:
Yeah it's unfortunate. I'm not happy with `undef`'ing things in Cesium Native so maybe renaming is a better choice. We already do that already for properties that conflict with reserved cpp words, like "class" becomes "clasProperty". Maybe "OPAQUE" could become "OPAQUE_ENUM".

Reply:
Updated the code